estate/models/estate_property.py

```diff
@@ -36,3 +37,46 @@ class EstateProperty(models.Model):
         copy=False,
         default="new",
     )
+    property_type = fields.Many2one("estate.property.type")
```

Comment:
Please follow guidelines to naming the fields https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#38fields

```suggestion
    property_type_id = fields.Many2one("estate.property.type")
```